src/components/AccessibilityGuide/AriaLandmarks.mdx

```diff
@@ -0,0 +1,119 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as SearchBarStories from "../SearchBar/SearchBar.stories";
+import Link from "../Link/Link";
+import Banner from "../Banner/Banner";
+
+<Meta title="Accessibility Guide/ARIA Landmarks" />
+
+# ARIA Landmarks
+
+## Table of Contents
+
+- {<Link href="#Landmarks-overview" target="_self">Landmarks overview</Link>}
+- {<Link href="#banner" target="_self">banner</Link>}
+- {<Link href="#complementary" target="_self">complementary</Link>}
+- {<Link href="#contentinfo" target="_self">contentinfo</Link>}
+- {<Link href="#form" target="_self">form</Link>}
+- {<Link href="#main" target="_self">main</Link>}
+- {<Link href="#navigation" target="_self">navigation</Link>}
+- {<Link href="#region" target="_self">region</Link>}
+- {<Link href="#search" target="_self">search</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## Landmarks overview
+
+Landmarks are a useful tool for people who use screen readers to orient themselves to important areas of a web page, and help quickly navigate to these areas within a website or application.
+
+ARIA’s eight landmark roles are specifically meant to describe page organization and structure, providing keyboard navigation to each larger section. Each of these roles has a corresponding native HTML element:
+
+| Landmark role | HTML element |
+| ------------- | ------------ |
+| banner        | `<header>`   |
+| complementary | `<aside>`    |
+| contentinfo   | `<footer>`   |
+| form          | `<form>`     |
+| main          | `<main>`     |
+| navigation    | `<nav>`      |
+| region        | `<section>`  |
+| search        | `<search>`   |
+
+Landmarks can be indicated with `role="landmark"` or by using the landmark's HTML element.
```

Comment:
Is one of these preferred or does it not matter?

Reply:
Maybe add something about how landmarks may be nested within one another and there are examples of that below (ie, within the `Header`)?